testcases/OpTestEEH.py

```diff
@@ -207,12 +241,26 @@ def check_eeh_slot_resets(self):
     # @returns True/False @type boolean
     #
     def check_eeh_pe_recovery(self, pe):
-        list = self.get_list_of_pci_devices()
-        for device in list:
-            if pe in device:
-                return True
-        return False
-
+     #   cmd = "dmesg  | grep -i 'iommu: Removing device'; echo $?"
+     #   tries = 30
+     #   for i in range(1, tries+1):
+     #       res = self.cv_SYSTEM.sys_get_ipmi_console().run_command(cmd)
+     #       if int(res[-1]):
+     #           print "Waiting for PE %s removal: (%d/%d)" % (pe, i, tries)
+     #           time.sleep(1)
+     #       else:
+	 #		    break
+     #   else:
+     #       raise EEHRemoveFailed("PE", pe)
+        tries = 60
```

Comment:
May be 60 is not needed, but we need multiple tries, as EEH sleeps for 5sec.

[57711.133255] iommu: Removing device 0002:01:00.3 from group 12
[57711.133348] iommu: Removing device 0002:01:00.2 from group 12
[57711.133437] iommu: Removing device 0002:01:00.1 from group 12
[57711.133513] iommu: Removing device 0002:01:00.0 from group 12
[57715.238340] EEH: Sleep 5s ahead of complete hotplug